src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
```

Comment:
If you are not using nested classes it is better to have a separate file for each class. 
This is a place where nested classes are ok.